blueprint/index.md

```diff
@@ -121,10 +121,10 @@ node server
 4. Click the **Configuration Tab**. If you're testing the project out of the box, type in the URL:
 
 ```
-https://localhost/?conversationid={pcConversationId}&language={pcLangTag}&environment={pcEnvironment}
+https://localhost/?conversationid={{pcConversationId}}&language={{pcLangTag}}&environment={{pcEnvironment}}
```

Comment:
Can't have double curlys in this content. This is reserved syntax for templates. They need to be escaped here.

Reply:
Cool.  That is what I wanted you to take a look at it.  Jerome had flagged this an error in his ticket and it did not seem like it, but glad you caught it.